CMakeLists.txt

```diff
@@ -1,3 +1,26 @@
+#
+#   This file is part of:
+#
+#   CTBignum 
+#
+#   C++ Library for Compile-Time and Run-Time Multi-Precision and Modular Arithmetic
+#   
+#   See the NOTICE file for the complete list of authors and contributors 
+#
+#   Copyright 2018 Niek J. Bouman
```

Comment:
This should read Copyright 2018 Authors (see NOTICE) or something like that. 
I would simply refer to the LICENSE file, according to DRY principle ;-)

Reply:
I've made a new pull request for this.